src/util/descriptionParser.tsx

```diff
@@ -0,0 +1,47 @@
+import { createElement, ReactNode } from "react";
+
+const RAW_DESCRPTION_TAG_NAME_MAP: { [k: string]: string } = { b: "strong", i: "em" };
+
+export function rawDescriptionToReact(rawDescription: string): ReactNode[] {
+  return rawDescriptionToReactRecursive(rawDescription).nodes;
+}
+
+function rawDescriptionToReactRecursive(rawDescription: string): {
+  nodes: ReactNode[];
+  stringLeft: string;
+} {
+  const nodes: ReactNode[] = [];
+  let stringLeftToProcess: string = rawDescription;
+  let processedString: string = "";
+
+  for (let i = processedString.length; i < stringLeftToProcess.length; i++) {
+    processedString += stringLeftToProcess[i];
+    const stringChunk = processedString.slice(processedString.length - 3);
+
+    //See if we're dealing with a tag, if not continue to process string
+    if (stringChunk.startsWith("</") || (stringChunk.startsWith("<") && stringChunk.endsWith(">"))) {
+      //We're dealing with luke's crazy paragraph things
+      if (stringChunk === "</p") {
+        nodes.push(processedString.slice(0, processedString.length - 3), createElement("p"));
```

Comment:
Is this correct? I think we are creating `p` nodes without any content in them. Edit: if this is the way it is generated currently I think its fine to merge as-is and maybe fix up later